packages/pangea-node-sdk/src/services/authn/session.ts

```diff
@@ -25,7 +24,7 @@ export default class Session extends BaseService {
    * ```
    */
   invalidate(sessionID: string): Promise<PangeaResponse<{}>> {
-    return this.post("session/invalidate", { session_id: sessionID });
+    return this.post("v1/session/invalidate", { session_id: sessionID });
```

Comment:
@pangea-andrest should we update these to v2 endpoints instead of v1?

Reply:
Yes. I'll review it, thanks!